wagtail/wagtailadmin/views/pages.py

```diff
@@ -678,7 +679,9 @@ def search(request):
         if form.is_valid():
             q = form.cleaned_data['q']
 
-            pages = Page.objects.all().prefetch_related('content_type').search(q, fields=['title'])
+            search_fields = getattr(settings, 'WAGTAIL_ADMIN_PAGE_SEARCH_FIELDS', ['title'])
```

Comment:
``WAGTAIL_ADMIN`` should be ``WAGTAILADMIN`` (for consistency with other settings)